2021-08-30 16:38:23 PM - WARNING - utils: Error occurred during loading data. Trying to use cache server https://fake-useragent.herokuapp.com/browsers/0.1.11
Traceback (most recent call last):
  File "C:\Python38\lib\site-packages\fake_useragent\utils.py", line 154, in load
    for item in get_browsers(verify_ssl=verify_ssl):
  File "C:\Python38\lib\site-packages\fake_useragent\utils.py", line 99, in get_browsers
    html = html.split('<table class="w3-table-all notranslate">')[1]
IndexError: list index out of range
2021-08-30 16:38:24 PM - INFO - agent: Response acquired
2021-08-30 16:38:32 PM - INFO - logger: Current Page: 0;Number of jobtype posts fetched in current page : 15;Number of jobtype posts fetched in total : 15;Number of jobtype posts remained to fetched : 3193
2021-08-30 16:38:32 PM - INFO - logger: Pause for 144
2021-08-30 16:40:57 PM - INFO - agent: Response acquired
